src/app/utils/device-characteristics-telemetry.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { getDeviceScreenScale, getBrowserScreenSize } from './device-characteristics-telemetry'
+
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation(query => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // Deprecated
+    removeListener: jest.fn(), // Deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn()
+  }))
+});
+describe('Device Telemetry', () => {
+  it('should get device screen scale', () => {
+    expect(getDeviceScreenScale()).toBeTruthy();
```

Comment:
There's formatting happening inside this function, I'd like to see the test assert that the result is formatted correctly.
Think of the test as a catch for a future developer who might come along and remove the formatting and break reporting as a result, a truthy check would still pass even if the value returned changed from `'100%'` to `1`